AWS/Page_AWS CloudFront.json

```diff
@@ -115,6 +115,17 @@
   },
   "sf_discoverySelectors" : [ "_exists_:DistributionId" ],
   "sf_description" : "",
+  "sf_filterAlias" : {
+    "variables" : [ {
+      "dimension" : "DistributionId",
+      "alias" : "DistributionId",
+      "globalScope" : true,
+      "required" : true,
+      "description" : "",
```

Comment:
Please add a description, this will be useful when users hover over the variable. Same for the variable on the EBS page.